src/ui/application-window.vala

```diff
@@ -592,6 +609,16 @@ namespace Peek.Ui {
         window_region.union (pop_format_region);
       }
 
+      //menu popover
+      if (pop_menu.visible) {
+        var pop_style = pop_menu.get_style_context ();
+        if (DesktopIntegration.get_theme_name () == "Ambiance") {
```

Comment:
Apps shouldn't have specific themes patches. If it's an issue upstream, it should be fixed there.

Reply:
Theme specific style removed.